examples/sdk/browser-keysigner/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "sdk-browser-keysigner-example",
```

Comment:
does it need to be a separate package or can this just go into the keysigner package? given it has 0 external lib dependency?